src/resourcemgr/scss/resourcemgr.scss

```
@@ -30,7 +30,49 @@
+        padding: 8px 5px;
+        border-bottom: 1px solid $uiGeneralContentBorder;
+        background-color: $uiGeneralContentBg;
+
+        &.hidden,
+        &[hidden] {
+            display: none;
+        }
+
+        .asset-search-label {
+            @include font-size(12);
+            margin: 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)